packages/base/src/dialogs/symbology/vector_layer/VectorRendering.tsx

```diff
@@ -37,7 +37,8 @@ const VectorRendering = ({
 
     const options: Record<string, string[]> = {
       VectorLayer: ['Single Symbol', 'Graduated', 'Categorized', 'Heatmap'],
-      VectorTileLayer: ['Single Symbol']
+      VectorTileLayer: ['Single Symbol'],
+      HeatmapLayer: ['Heatmap']
```

Comment:
Shouldn't this be:

```suggestion
      HeatmapLayer: ['Single Symbol', 'Graduated', 'Categorized', 'Heatmap']
```

Otherwise it seems we can't go back to a "single symbol" after setting heatmap rendering:


https://github.com/user-attachments/assets/855e69c9-b1c9-44ad-a274-16d611fef7c6

Reply:
Ahh yes, you're right.
I didn't test for this case, thank you for taking care of this.